### DISCUSSION TITLE
On/Off toggle's flag

### DISCUSSION THREAD

**Discussion by bob0dod** [mjakeman/extension-manager#621] [Q&A]
Does somebody know if (and where) this application saves the on/off preferences inside a configuration file?

**Comment by oscfdezdz** [comment]
No, it doesn't, but you can get them by either running `gnome-extensions` tool or `gsettings` command in a terminal.
- gnome-extensions: `gnome-extensions info <extension@uuid>`. See `gnome-extensions help` for all available commands.
- gsettings: `gsettings get org.gnome.shell enabled-extensions` for enabled extensions and `gsettings get org.gnome.shell disabled-extensions` for disabled ones.
